### PR TITLE
[skip ci] Update readme and contributing doc

### PR DESCRIPTION
**Pull request by anchal-agrawal**
This commit polishes the VIC Engine readme and contributing markdown
files and highlights some new key features in the readme's Project
Status section.

Towards #5668

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,69 +6,63 @@ vSphere Integrated Containers Engine (VIC Engine) is a container runtime for vSp
...
+[wizard]:https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/deploy_vch_client.html
+[dch]:https://vmware.github.io/vic-product/assets/files/html/1.3/vic_vsphere_admin/deploy_vch_dchphoton.html
+
 VIC Engine now provides:
-* basic function for most of the core lifecycle operations: pull, create, start, attach, run, kill, stop, wait, rm, logs.
-* early vCenter support, leveraging DRS for initial placement.
-* volume support [--volume-store](doc/user/usage.md#configuring-volumes-in-a-virtual-container-host) - SIOC is not integrated but can be set as normal.
+* support for most of the Docker commands for core container, image, volume and network lifecycle operations. Several `docker compose` commands are also supported. See the complete list of supported commands [here][commands].
+* vCenter support, leveraging DRS for initial placement. vMotion is also supported.
+* volume support for vSphere datastores (including vSAN) and NFS shares. See [--volume-store](doc/user/usage.md#configuring-volumes-in-a-virtual-container-host) - SIOC is not integrated but can be set as normal.
```
Comment (anchal-agrawal) [inline README.md]: Ping @matthewavery for reviewing this part.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -191,72 +193,75 @@ priority/p3 - Bugs that SHOULD be fixed by a given release, time permitting.
 
 priority/p4 - Bugs that SHOULD be fixed in a future (to be determined) release.
 
-### Backlog
+### Not Ready
```
Comment (anchal-agrawal) [inline CONTRIBUTING.md]: Ping @mhagen-vmware for reviewing these sections - these changes add details and guidelines about our pipeline lanes. The recent priority changes are unaffected.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -191,72 +193,75 @@ priority/p3 - Bugs that SHOULD be fixed by a given release, time permitting.
...
+4. no assignee (assignees are set when the issue is selected to work on)
+
+Story Points | Story Size
+------------ | -------------------------------------------------------
+1            | Less than 1 day of effort
+2            | 2 - 3 days of effort
+3            | 3 - 5 days of effort
+5            | 5 - 10 days of effort, consider splitting this if it's 7 - 10 days
+8            | > 10 days, anything of this size should be split before moving into In Progress
+
```
Comment (mhagen-vmware) [inline CONTRIBUTING.md]: This is fundamentally not a good idea for several reasons, we should stick with our existing definitions of story points or we are not using story points at all and need to move to time based estimates ( which aren't good, thus Agile uses story points )

Comment (anchal-agrawal) [reply CONTRIBUTING.md]: I see. My change doesn't add this table, it's already there: https://github.com/vmware/vic/blob/master/CONTRIBUTING.md#in-progress. I've just moved it from the In Progress section to the Backlog section.

Comment (anchal-agrawal) [reply CONTRIBUTING.md]: Should we remove this table altogether?

Comment (mhagen-vmware) [reply CONTRIBUTING.md]: I would like to see it corrected to our agreed upon estimation process:
```
Story points
● We have not been consistent in story point estimation
● Proposal:
– Stick with fibonacci
– All bugs are a 2 unless we know it is significantly more or less work than the
average bug
– 1 is easier than the average bug
– 3 is slightly more work than the average bug and probably should be about
an average feature work for an easy feature (which includes design doc,
implementation, testing, review)
– 5 is about 2x more work than the average bug and the highest single issue
value we want
– >5 can be on EPICs, but single issues should be decomposed further if you
think they are more than a 5
```

Comment (anchal-agrawal) [reply CONTRIBUTING.md]: Fixed: https://github.com/anchal-agrawal/vic/blob/860c9b267b8c59d9dc6057613bfc982213a787e7/CONTRIBUTING.md#story-point-estimates

File: README.md (hunk 1 above)
Comment (matthewavery) [reply README.md]: is the [here][commands] mean to be a hyperlink? if so It might be missing the link :)

Comment (matthewavery) [reply README.md]: `vsphere datastores` may not be quite correct. we do support standard vmfs backed datastores. We do support vsan datastores, NFS datastores, and ISCSI datastores. However, things like VVOL datastores and any other kind of specialty datastores are not supported. :)

Comment (anchal-agrawal) [reply README.md]: The `[commands]` hyperlink is defined here: https://github.com/vmware/vic/pull/6974/files#diff-04c6e90faac2675aa89e2176d2eec7d8R11

**Pull request merged**